lib/primitives/block.js

```diff
@@ -247,7 +247,7 @@ class Block extends AbstractBlock {
     const witnessRoot = this.createWitnessRoot();
 
     if (!witnessRoot.equals(this.witnessRoot))
-      return [false, 'bad-txnmrklroot', 100];
+      return [false, 'bad-witnessroot', 100];
```

Comment:
This is same as `bad-witness-merkle-match` in btc I believe. (even though they don't serialize it with block)

Reply:
Yeah I saw that. I think when I wrote this I wanted a variation of `bad-txnmrklroot` (used by hsd and bitcoin core) since yeah our witness root is in the block header and is called `witnessRoot` all over the codebase.